wiki/osu!_File_Formats/Osu_(file_format)/pt-br.md

```diff
@@ -0,0 +1,358 @@
+
+### Spinners
+
+*Sintaxe dos spinners:* `x,y,time,type,hitSound,endTime,hitSample`
+
+- **`endTime` (Inteiro):** Tempo do fim do spinner, em milissegundos a partir do começo do audio do beatmap.
+- `x` e `y` não afetam spinners. Seu padrão é o centro do campo de jogo, `256,192`.
+
+### Holds (apenas osu!mania)
+
+*Sintaxe do hold:* `x,y,time,type,hitSound,endTime:hitSample`
+
+- **`endTime` (Inteiro):** Tempo do fim do hold, em milissegundos a partir do começo do áudio do beatmap.
+- `x` determina o índice da coluna que o hold estará. Isso é computado com `floor(x * columnCount / 512)` e arredondado para entre `0` e `columnCount - 1`.
+- `y` não afeta holds. Seu padrão é o centro do campo de jogo, `192`.
+
+### Exemplos
+
+```
+256,192,11000,21,2
+256,192,11200,8,12,12000,3:0:0:80:
+100,100,12600,6,1,B|200:200|250:200|250:200|300:150,2,310.123,2|1|2,0:0|0:0|0:2,0:0:0:0:
+```
+
+O primeiro objeto é um hit circle:
+
+- No centro da tela
+- Em 11 segundos
+- Começando um novo combo, e pulando uma cor extra de combo
+- Com um hitsound whistle
+
+O segundo objeto é um spinner:
+
+- Em 11.2 segundos
+- Acabando em 12 segundos
+- Com hitsounds finish e clap, tocando no volume 80%
+- Com o hitsound normal tocando com o drum set, no volume 80%
+
+O terceiro objeto é um slider:
+
+- Na posição (100,100)
+- Em 12.6 segundos
+- Começando um novo combo
+- Com o corpo do slider sendo uma curva de bézier composta, onde os pontos de controle da primeira curva são (100,100), (200,200) e (250,200), e os pontos de controle da segunda curva são (250,200), (300,150)
+- Repetindo uma vez
+- 310.123 osu! pixels de comprimento
+- Com um hitsound whistle no início, e um hitsound whistle tocando com o soft set no final
+
+### osu!taiko
+
+Hit objects no osu!taiko utilizam apenas `time` para determinar como eles são colocados no campo de jogo, então `x` e `y` são ignorados.
+osu!taiko's hit objects only use `time` to determine how they are placed on the play field, so `x` and `y` are ignored. A única parte significante das curvas do slider é `length`; `curveType` e `curvePoints` são relevantes apenas ao abrir o mapa no editor. Cores de combo e novos combos são ignorados, e hitsounds específicos para cada modo são utilizados
+
+- Hit circles com hitsounds whistle ou clap se tornam kats, e outros hit circles se tornam dons. Adicionar o finish hitsound muda esses em suas grandes variantes.
+- Sliders se tornam drum rolls.
+- Spinners se tornam denden notes.
+
+### osu!catch
+
+O campo de jogo do osu!catch utiliza apenas o eixo x, então `y` é irrelevante. Curvas de slider podem utilizar espaço vertical para alcançar aceleração horizontal quando eles são achatados para um campo de jogo de uma dimensão.
+
+- Hit circles se tornam fruits.
+- Sliders se tornam juice streams, com fruits em cada canto.
```

Comment:
I thought about translating `fruits` as `frutas` since it's a common word that describe exactly what it is but `juice streams` does not have a nice translation so I leave it like this because I haven't found any osu!catch article translated.